draft-ietf-quic-ack-frequency.md

```diff
@@ -451,11 +451,13 @@ To limit the consequences of reduced acknowledgement frequency, a sender
 SHOULD cause a receiver to send an acknowledgement at least once per RTT if
 there are unacknowledged ack-eliciting packets in flight.
 
-A sender can accomplish this by setting the Ack-Eliciting Threshold
-to a value no larger than the current congestion window or the Request Max Ack
-Delay value to no more than the estimated round trip. Alternatively,
-a sender can accomplish this by sending an IMMEDIATE_ACK frame once per
-round trip, though if the packet containing an IMMEDIATE_ACK is lost,
+A sender can accomplish this by setting the Request Max Ack
+Delay value to no more than the estimated round trip time.
+The sender can also improve feedback and robustness to
+variation in the path RTT by setting the Ack-Eliciting Threshold
+to a value no larger than the current congestion window. Alternatively,
+a sender can accomplish this by sending an IMMEDIATE_ACK frame once each
+round trip time or after each application-limited period, although if the packet containing an IMMEDIATE_ACK is lost,
```

Comment:
What is the "application-limited period"? I assume you mean when transitioning from the application-limited to the cwnd-limited state?

Reply:
Actually, good point. Do we need this addition? Even in an application-limited period, you should send on ack per RTT. Why is this needed? Or do you mean idle period @gorryfair ?

Reply:
(This could potentially be confusing as an "application period" since "period of time" could infer periodicity - English can be used in multiple ways.) 

That aside, the guidance in the application-limited case ought to be to send an ACK at least once per RTT while there is data outstanding... It's a small additional tweak to request an Immediate ACK as soon as the sender transitions from being application-limited to become cwnd-limited, so that the window is released promptly, rather than delayed by upto an RTT.

Obviously (to me) that when idle, you don't request periodic ACKs....